docs/start/getting-started/fragments/android/generate-model.md

```diff
@@ -30,6 +30,11 @@ With the basic setup complete, next you will model the data your application wil
     - **priority** an optional enumeration type field that indicates the importance of a Todo item; the value of priority can be only *LOW*, *NORMAL*, or *HIGH*
     - **description** an optional string field that holds more information about a Todo item
 
+You can optionally add a plugin for GraphQL syntax highlighting by going to Android Studio > Preferences and go to the Plugins tab. Then you can search the Plugin Marketplace for the JS GraphQL plugin. Afterwards, install and activate the plugin, you may need to restart IDE.
+
+![](~/images/lib/getting-started/android/set-up-android-studio-install-plugin.png)
```

Comment:
This is a lot of friction for not much value during a getting starting guide. Let's move this somewhere else - say Project Setup.